### PR TITLE
Add show replicas DAL SQL parser for MySQL.

### LINKED ISSUE #13135: support parsing SHOW REPLICAS for mysql
Hi community,

This issue can be a good first try.

## Aim
support parse mysql administration statements sql `SHOW REPLICAS`

## Basic Qualifications
- java
- maven
- antlr4

## Detail
Add antlr rule in DALStatement.g4. refer to https://dev.mysql.com/doc/refman/8.0/en/show-replicas.html
Implement the visit method in `shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/impl/MySQLDALStatementSQLVisitor.java`
To complete this issue, I think we should complete these steps:
- Create anltr rule in DALStatement.g4
- Create `MySQLShowReplicasStatement` class in this package `org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.dal`
- Add rule in `shardingsphere-sql-parser/shardingsphere-sql-parser-engine/src/main/java/org/apache/shardingsphere/sql/parser/core/database/visitor/SQLVisitorRule.java`
- Implement the visit method in `shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/impl/MySQLDALStatementSQLVisitor.java`. 
- Finally, add test for parse clone statement in `shardingsphere-test` module
## Example FYI
- Refer to this merged pr：https://github.com/apache/shardingsphere/pull/12979/files

### PR DESCRIPTION
**Pull request by dongzl**
Fixes #13135 .

Changes proposed in this pull request:
- Add show replicas antlr grammar for MySQL.
- Add show replicas DAL SQL parser for MySQL.
- Add show replicas DAL SQL parser test case for MySQL.

### CODE REVIEW

**Comment by strongduanmu** [comment]
Hi @dongzl, welcome back first, and can you fix ci error?

**Comment by dongzl** [comment]
> Hi @dongzl, welcome back first, and can you fix ci error?

OK, I will fix it.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4 [hunk 1]
```diff
@@ -246,6 +246,10 @@ showWarnings
     : SHOW (COUNT LP_ ASTERISK_ RP_)? WARNINGS (LIMIT (NUMBER_ COMMA_)? NUMBER_)?
     ;
 
+showReplicas
+    :SHOW REPLICAS
```
Comment (strongduanmu) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4:250]: A space is required after the colon.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/mysql/dal/MySQLShowReplicasStatement.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import lombok.ToString;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.dal.DALStatement;
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.MySQLStatement;
+
+/**
+ * MySQL show replicas statement.
+ */
+@ToString
+public class MySQLShowReplicasStatement extends AbstractSQLStatement implements DALStatement, MySQLStatement {
```
Comment (strongduanmu) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/mysql/dal/MySQLShowReplicasStatement.java:29]: @dongzl Please add final for MySQLShowReplicasStatement.

File: shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/cases/domain/statement/dal/ShowReplicasStatementTestCase.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+package org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.statement.dal;
+
+import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.statement.SQLParserTestCase;
+
+/**
+ * Show replicas test case.
+ */
+public class ShowReplicasStatementTestCase extends SQLParserTestCase {
```
Comment (strongduanmu) [inline shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/cases/domain/statement/dal/ShowReplicasStatementTestCase.java:25]: Please add final here.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4 (hunk 1 above)
Comment (dongzl) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4]: Thanks, fix it.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/mysql/dal/MySQLShowReplicasStatement.java (hunk 2 above)
Comment (dongzl) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/mysql/dal/MySQLShowReplicasStatement.java]: Thanks, fix it.

**Review by strongduanmu — APPROVED** [review]

File: shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/cases/domain/statement/dal/ShowReplicasStatementTestCase.java (hunk 3 above)
Comment (dongzl) [reply shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/cases/domain/statement/dal/ShowReplicasStatementTestCase.java]: Thanks, fix it.

**Pull request merged**